src/pages/filter/dtree/components/query-builder/ui/next-step/content-code/content-code.module.css

```diff
@@ -0,0 +1,36 @@
+.content-code {
+  display: flex;
+  height: auto;
+  width: 33.333%;
+  margin-left: 8px;
+
+  &_wrapper {
```

Comment:
Please, use double underscore '__' for elements, and single '_' for modifiers. Like, `.block__element_modifier`